docs/reference/controls/itemscontrol.md

```diff
@@ -118,7 +118,7 @@ The view resizes horizontally, but content is hidden when it is too high. This c
 <img src={example01Screenshot} alt="" />
 
 ## Example 02
-This example bind an observable collection of tile items to an items control. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
+This example binds an observable collection of Rectangles (which are bound to a tile VM) to an ItemsControl. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
```

Comment:
You are not positioning the tile, you are positioning the rectangle based on the title data. It's an important semantic distinction to make for this sort of thing and making that clear can help people who are not familiar with MVVM.